tests/test_ci/TestControllers.py

```diff
@@ -183,25 +183,53 @@ def test_customizedtest_added_to_queue(self):
         self.assertNotIn(1, customized_test)
 
     @mock.patch('mailer.Mailer')
-    def test_inform_mailing_list(self, mock_email):
+    @mock.patch('mod_ci.controllers.get_html_issue_body')
+    def test_inform_mailing_list(self, mock_get_html_issue_body, mock_email):
         """
         Test the inform_mailing_list function
         """
         from mod_ci.controllers import inform_mailing_list
-        from mailer import Mailer
 
+        mock_get_html_issue_body.return_value = """2430 - Some random string\n\n
+                     Link to Issue: https://www.github.com/test_owner/test_repo/issues/matejmecka\n\n
+                     Some random string(https://github.com/Some random string)\n\n\n
+                     Lorem Ipsum sit dolor amet...\n        """
         email = inform_mailing_list(mock_email, "matejmecka", "2430", "Some random string",
                                     "Lorem Ipsum sit dolor amet...")
 
         mock_email.send_simple_message.assert_called_once_with(
             {
-                'text': '2430 - Some random string\n\n'
-                '        Link to Issue: https://www.github.com/test_owner/test_repo/issues/matejmecka\n\n'
-                '        Some random string(https://github.com/Some random string)\n\n\n'
-                '        Lorem Ipsum sit dolor amet...\n        ',
-                'subject': 'GitHub Issue #matejmecka', 'to': 'ccextractor-dev@googlegroups.com'
+                'to': 'ccextractor-dev@googlegroups.com',
+                'subject': 'GitHub Issue #matejmecka',
+                'html': """2430 - Some random string\n\n
+                     Link to Issue: https://www.github.com/test_owner/test_repo/issues/matejmecka\n\n
+                     Some random string(https://github.com/Some random string)\n\n\n
+                     Lorem Ipsum sit dolor amet...\n        """
             }
         )
+        mock_get_html_issue_body.assert_called_once()
+
+    @staticmethod
+    @mock.patch('mod_ci.controllers.markdown')
+    def test_get_html_issue_body(mock_markdown):
+        """
+        Test the get_html_issue_body for correct email formatting
+        """
+        from mod_ci.controllers import get_html_issue_body
+
+        title = "[BUG] Test Title"
+        author = "abcxyz"
+        body = ("**strong_text**<br/>*em_text*\n\n"
```

Comment:
If we don't care about the contents here, I don't see a reason to have the multiline body :)